### PR TITLE
Create `.upgrade-config.yml` for each provider repo

### PR DESCRIPTION
**Pull request by aq17**
Once https://github.com/pulumi/upgrade-provider/pull/66 is merged (specifically [this commit](https://github.com/pulumi/upgrade-provider/pull/66/commits/ee97107c5b53f347b8fee34d57ad094abd2f9b0b)), we initialize a config file for each provider, with only `upstream-provider-name` set for now

### CODE REVIEW

**Comment by iwahbe** [comment]
Could we do this by defining the config file consumed in https://github.com/pulumi/upgrade-provider/pull/61 for each provider? That way, it remains easy to run the `upgrade-provider` tool locally.

**Comment by aq17** [comment]
> Could we do this by defining the config file consumed in [pulumi/upgrade-provider#61](https://github.com/pulumi/upgrade-provider/pull/61) for each provider? That way, it remains easy to run the `upgrade-provider` tool locally.

Done 👍🏼

File: provider-ci/src/provider.ts [hunk 1]
```diff
@@ -127,3 +132,14 @@ export function generateProviderFiles(config: BridgedConfig) {
...
+export function getUpstreamProviderName(providerName: string) {
+	let nameMap = new Map<string, string>([
+		["f5bigip", "bigip"],
+		["confluentcloud", "confluent"]
+	])
+	if (nameMap.has(providerName)) {
+		providerName = nameMap.get(providerName)!
+	}
+	return "terraform-provider-" + providerName
+}
```
Comment (iwahbe) [inline provider-ci/src/provider.ts:145]: This information already exists in provider config.

For example:
https://github.com/pulumi/ci-mgmt/blob/d85f5e6d456856652bc708e69ffc48bba5e7ce72/provider-ci/providers/confluentcloud/config.yaml#L8

**Review by iwahbe — APPROVED** [review]
Lets make sure we don't merge until our GH action is using a version of `upgrade-provider` that can consume these files.

I believe that this will also require bumping the version of our new CI action that these providers depend on.

**Review by t0yv0 — APPROVED** [review]
Looks clean.

**Pull request merged**